e2e/runner/cypress-runner-generate-snapshots.js

```diff
@@ -11,7 +13,14 @@ const getConfig = baseUrl => {
 };
 
 const generateSnapshots = async (baseUrl, exitFunction) => {
-  const snapshotConfig = getConfig(baseUrl);
+  // We only ever care about a broswer out of all possible user arguments,
+  // when it comes to the snapshot generation.
+  // Anything else could result either in a failure or in a wrong database snapshot!
+  const { browser } = await parseArguments(args);
+  const customBrowser = browser ? { browser } : null;
+
+  const baseConfig = getConfig(baseUrl);
+  const snapshotConfig = Object.assign({}, baseConfig, customBrowser);
```

Comment:
you don't really need to change this if you want to avoid running the CI checks again. But FE engineers usually use an object destructure syntax instead.

```suggestion
  const snapshotConfig = { ...baseConfig, ...customBrowser };
```

I personally like using a merge util, but it needs another import e.g. either from `icepick` or `underscore`, so I also stick to the object destructuring syntax.

```suggestion
  const snapshotConfig = merge(baseConfig, customBrowser);
```